File: ass65/Program1.java
/*
1. Write a java program which accept number of rows and number of columns from user and display below pattern. 
Input :  iRow = 4  iCol = 4  
Output : * * * #   
         * * # *    
         * # * * 
         # * * * 
*/
import java.lang.*;
import java.util.*;

class Matrix 
{
    void Row(int arr[][],int row,int col)
    {
        
        
        for( int i=0; i<row; i++)
        {
            for(int j =0; j<col; j++)
            {
                if((i+j)==(row-1))
                {
                    System.out.print("#"); 
                }
                else 
                {
                    System.out.print("*"); 
                }
            }
            System.out.println("");
        }
    }
}
class Program1
{
	public static void main(String arg[])
	{
		Scanner sobj = new Scanner(System.in);
		System.out.println("Enter number of rows");
		int row = sobj.nextInt();

		System.out.println("Enter number of columns");
        int col = sobj.nextInt();

		int arr[][] = new int[row][col];

        Matrix dobj = new Matrix();
        dobj.Row(arr,row,col);
	}
}

File: ass12/program1.c
/*
Write a program which accept number from user and display below pattern. 
Input :  5  Output :  * * * * * # # # # #   
Input :  6 Output :  * * * * * * # # # # # # #   
Input :  -5  Output :  * * * * * # # # # #   
Input :  2 Output :  * * # #
*/
#include<stdio.h>
void Display(int iNo)
{
    int iCnt=0;
    if(iNo<0)
    {
        iNo=-iNo;
    }
    for (iCnt=1;iCnt<=iNo;iCnt++)
    {
        printf("*\n");
    }
     for (iCnt=1;iCnt<=iNo;iCnt++)
    {
        printf("#\n");
    }
    
}
int main()
{
    int iValue=0;
    printf("enter the element\n");
    scanf("%d",&iValue);
    Display(iValue);
    return 0;
}